drivers/amazonec2/amazonec2.go

```diff
@@ -664,42 +658,52 @@ func (d *Driver) configureSecurityGroup(groupName string) error {
 	return nil
 }
 
-func configureSecurityGroupPermissions(group *amz.SecurityGroup) []amz.IpPermission {
+func (d *Driver) configureSecurityGroupPermissions(group *amz.SecurityGroup) []amz.IpPermission {
```

Comment:
Ah, the test needs to be updated to reflect that this has been moved to a driver method.